src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/SpawnObject.java

```diff
@@ -71,8 +71,8 @@ public int getId() {
 		
 		public RecyclableArrayList<ClientBoundPacketData> updateItem(ProtocolVersion version, ItemStackWrapper itemstack) {
 			RecyclableArrayList<ClientBoundPacketData> updatepackets = RecyclableArrayList.create();
-			if(this.itemstack != itemstack) {
-				if(spawned) updatepackets.add(getDespawnPacket(version));
+			if (this.itemstack != itemstack) {
+				if (spawned) {updatepackets.add(getDespawnPacket(version));}
```

Comment:
I will do it, but sometimes, unless you're trying to write a book it will become very clogged if every short if takes up three lines..